src/model.rs

```diff
@@ -209,16 +226,248 @@ impl QueryResponse {
         }
     }
 
+    pub fn get_affected_rows(&self) -> u32 {
+        self.affected_rows
+    }
+
     pub fn __str__(&self) -> String {
         format!("{:?}", self)
     }
 }
 
-pub fn convert_queried_rows(queried_rows: QueriedRows) -> PyResult<QueryResponse> {
+pub fn convert_query_response(query_resp: RustQueryResponse) -> PyResult<QueryResponse> {
     Ok(QueryResponse {
         schema: Schema {
-            raw_schema: Arc::new(queried_rows.schema),
+            raw_schema: Arc::new(query_resp.schema),
         },
-        raw_rows: Arc::new(queried_rows.rows),
+        raw_rows: Arc::new(query_resp.rows),
+        affected_rows: query_resp.affected_rows,
     })
 }
+
+/// Value in local, define to avoid using the one in ceresdb.
+#[pyclass]
+#[derive(Clone, Debug)]
+pub struct Value {
+    raw_val: rust_model::value::Value,
+}
+
+#[pyclass]
+#[derive(Clone, Debug, Default)]
+pub struct ValueBuilder;
+
+#[pymethods]
+impl ValueBuilder {
+    #[new]
+    pub fn new() -> Self {
+        Self::default()
+    }
+
+    pub fn with_i8(&self, val: i8) -> Value {
```

Comment:
I'm not sure whether we should remove the prefix `with_`? Is there any reference suggesting this style in python? I guess we'd better remove the prefix if not.

Reply:
I see. The method names may be a conflict with rust's keywords if remove the prefix.